pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -568,6 +572,10 @@ func (p *PriorityQueue) movePodsToActiveOrBackoffQueue(podInfoList []*framework.
 				klog.ErrorS(err, "Error adding pod to the scheduling queue", "pod", klog.KObj(pod))
 			} else {
 				metrics.SchedulerQueueIncomingPods.WithLabelValues("active", event.Label).Inc()
+				var val = pInfo.UnschedulablePlugins.List()
+				for _, plugin := range val {
+					metrics.UnschedulableReason(plugin, pod.Spec.SchedulerName).Dec()
```

Comment:
We should only decrease the metric when the Pod is getting tried, so in `scheduleOne` perhaps. But also you need to take care of deleted pods.

Reply:
Or maybe in `queue.Pop`